pkg/health/bird/bird.go

```diff
@@ -39,6 +39,10 @@ func GRInProgress(ipv string) (bool, error) {
 	// Try connecting to the BIRD socket in `/var/run/calico/` first to get the data
 	c, err := net.Dial("unix", fmt.Sprintf("/var/run/calico/bird%s.ctl", birdSuffix))
 	if err != nil {
+		// If that fails and BIRD socket file exists, return the error message
+		if !isNotExistError(err) {
```

Comment:
does `os.IsNotExist()` work for the errors returned by `net.Dial`?

Reply:
Thanks,i will use os.IsNotExist to replace the checking error text.